scripts/jenkins/jobs-ibs/cloud-mkcloud6-job-hyperv.yaml

```diff
@@ -0,0 +1,25 @@
+- job:
+    name: 'cloud-mkcloud6-job-hyperv'
+    node: cloud-trigger
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud6
+            nodenumber=3
+            networkingplugin=openvswitch
+            networkingmode=vlan
+            libvirt_type=hyperv
+            mkcloudtarget=all
```

Comment:
actually, that is not needed, because of the libvirt_type=hyperv causing this OS selection.